content/10.diagnostics.md

```diff
@@ -481,14 +484,14 @@ In some cases, clinical signs may also provide information that can inform diagn
 Using computed tomography of the chest in addition to RT-qPCR testing was found to provide a higher sensitivity than either measure alone [@doi:10.1101/2020.02.25.20027755].
 X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies [@doi:10.1148/radiol.2020200642].
 Other studies have shown that specificity varies between radiologists [@doi:10.1148/radiol.2020200823], though the sensitivity reported here was lower than that published in the previous paper.
-However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination [@doi:10.1007/s13246-020-00865-4].
+However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination [@doi:10.1007/s13246-020-00865-4]. <!-- Do we take these results at face value? Studies like https://doi.org/10.1038/s42256-021-00307-0 have questioned whether ML models like this have any clinical value. I'm not sure whether the surveyed this particular paper we cite here, but some additional caveats or skepticism may be needed (in the next sentence too). -->
```

Comment:
Strongly agreed; if we decide to mention ML models here they should be discussed a bit more thoroughly.

Reply:
@mprobson any interest?

Reply:
Taking a look, I'm not sure I'm the right person to address the potential deficiencies in ML models for medicine but I agree some extra context and caveats are warranted.

Reply:
```suggestion
While preliminary machine-learning results suggested that chest X-rays might provide high sensitivity and selectivity and potentially facilitate the detection of asymptomatic and presymptomatic infections (e.g., [@doi:10.1007/s13246-020-00865-4]), further investigation suggested that such approaches are prone to bias and are unlikely to be clinically useful [@doi:10.1038/s42256-021-00307-0].